bin/remove_CatalogServiceZenPack.sh

```diff
@@ -18,7 +18,10 @@ if [[ ! $status -eq 0 ]]; then
     rm $tmp_output
     echo "Couldn't remove CatalogService zenpack"
     echo "You should remove CatalogService zenpack manually before starting the upgrade"
+    echo "Make sure you have Zenoss.resmgr/Infrastructure/mariadb-model, Zenoss.resmgr/Infrastructure/mariadb-events, Zenoss.resmgr/Infrastructure/RabbitMQ, Zenoss.resmgr/Zenoss/Events/zeneventserver, Zenoss.resmgr/Infrastructure/redis, Zenoss.resmgr/Infrastructure/memcached services running and Zenoss.resmgr/Infrastructure/zencatalogservice stoped, and then from your master host run command:"
+    echo "serviced service run zope zenpack-manager uninstall ZenPacks.zenoss.CatalogService"
```

Comment:
I think this needs reformatting to improve readability.  Here's an example output I, as an operator, would want to see:
```
Couldn't remove CatalogService ZenPack.
Please remove the CatalogService ZenPack manually before starting the upgrade.
To remove the CatalogService ZenPack:
1. Ensure the following services are running:
   * Zenoss.resmgr/Infrastructure/mariadb-model
   * Zenoss.resmgr/Infrastructure/mariadb-events
   * Zenoss.resmgr/Infrastructure/RabbitMQ
   * Zenoss.resmgr/Zenoss/Events/zeneventserver
   * Zenoss.resmgr/Infrastructure/redis
   * Zenoss.resmgr/Infrastructure/memcached
2. Ensure the following service has stopped:
   * Zenoss.resmgr/Infrastructure/zencatalogservice
3. Log onto the host the Control Center master and run the following command:
   serviced service run zope zenpack-manager uninstall ZenPacks.zenoss.CatalogService
```